packages/badge/src/Badge.ts

```diff
@@ -51,6 +51,21 @@ export class Badge extends SizedMixin(ObserveSlotText(SpectrumElement, '')) {
     public variant: BadgeVariant = 'informative';
 
     protected override render(): TemplateResult {
+        if (window.__swc.DEBUG) {
+            if (!BADGE_VARIANTS.includes(this.variant)) {
+                window.__swc.issueWarning(
+                    this,
+                    {
+                        localName: this.localName,
+                        type: 'api',
+                        level: 'default',
+                    },
+                    `<${this.localName}> element expect the "variant" attribute to be one of the following:`,
+                    'https://opensource.adobe.com/spectrum-web-components/components/badge/#variants',
```

Comment:
Since tooling takes care of quickly & reliably replacing strings, splitting a url into multiple parts like `${BASE}/badge/#variants` can make it harder to make updates when the url changes.

Another nice thing in vscode & other tools is that you can cmd+click on a full URL to open it.

Reply:
VSCode click to open is a really powerful point. I do like _less_, but I don't think this is a place where that's possible right now.